web/templates/submission/index.html

```diff
@@ -190,13 +194,14 @@
                                     </div>
                                 {% endif %}
                                 <div class="form-group">
-                                    <label for="form_package">Analysis Package</label>
+                                    <label for="form_package">Analysis Package (hover for more info)</label>
                                     <select class="form-control" id="form_package" name="package">
-                                        <option value="" active>Detect Automatically</option>
+                                        <option value="" title="">Detect Automatically</option>
                                         {% for package in packages %}
-                                        <option value="{{package}}">{{package}}</option>
+                                        <option value="{{package.value}}" title="{{ package.description }}">{{package.name}} - {{ package.summary }}</option>
```

Comment:
Here is where we display the description and the summary for each package in the list.